lib/ueberauth/strategy/amco.ex

```diff
@@ -202,13 +202,7 @@ defmodule Ueberauth.Strategy.Amco do
 
   defp fetch_user(conn, token) do
     conn = put_private(conn, :amco_token, token)
-    put_private(conn, :amco_user, %{
-      "sub" => "10",
-      "email" => "alejandrodevs@gmail.com",
-      "last_name" => "Gutierrez",
-      "first_name" => "Alejandro",
-      "phone_number" => "+523310971195"
-    })
+    put_private(conn, :amco_user, %{})
```

Comment:
I removed this dummy data that I was used for testing. This function should validate the id_token or consumes the `/userinfo` (it is not implemented yet) endpoint in the OIDC Provider.